### PR TITLE
Port Jeuno NPC Shemo

### PR DESCRIPTION
**Pull request by mikeyuzu**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/DerpyProjectGroup/topaz/blob/info/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/DerpyProjectGroup/topaz/blob/info/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Seal conversion

### CODE REVIEW

**Comment by Era-Lusiphur** [comment]
Tested this locally. Traded a few types back and forth. Appears to function well.

File: scripts/zones/Port_Jeuno/npcs/Shemo.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+require("scripts/globals/quests")
+require("scripts/globals/utils")
+-----------------------------------
+local entity = {}
+
+local beastman_seal = 0         -- �b�l���
+local kindred_seal = 1          -- �b�_���
+local kindred_crest = 2         -- ���l���
+local high_kindred_crest = 3    -- �������
+local sacred_kindred_crest = 4  -- ���_���
```
Comment (Xaver-DaRed) [inline scripts/zones/Port_Jeuno/npcs/Shemo.lua]: This comments were originaly in Japanese, I think, but they have lost format. Could you please change them to english or remove them?

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]
Only those 2 style changes. Other than that, welcome and thank you^^

File: scripts/zones/Port_Jeuno/npcs/Shemo.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+entity.onEventUpdate = function(player, csid, option)
+end
+
+entity.onEventFinish = function(player, csid, option)
+    if csid == 352 then
+        local addnum = bit.rshift(option, 8)
+        local out_seal = bit.rshift(bit.band(option, 0xFF), 4) - 1
+        local in_seal = bit.band(option, 0xF) - 1
+        local dellseals = addnum * 3
+        if (isTradeNum2 == 1) then
```
Comment (Xaver-DaRed) [inline scripts/zones/Port_Jeuno/npcs/Shemo.lua]: Please remove the braces here...
if isTradeNum2 == 1 then

File: scripts/zones/Port_Jeuno/npcs/Shemo.lua (hunk 1 above)
Comment (mikeyuzu) [reply scripts/zones/Port_Jeuno/npcs/Shemo.lua]: It has been deleted

File: scripts/zones/Port_Jeuno/npcs/Shemo.lua (hunk 2 above)
Comment (mikeyuzu) [reply scripts/zones/Port_Jeuno/npcs/Shemo.lua]: Removed curly braces

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/zones/Port_Jeuno/npcs/Shemo.lua [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+local kindred_crest = 2
+local high_kindred_crest = 3
+local sacred_kindred_crest = 4
+
+-- 0:Convert at 3: 1
+-- 1:Convert at 2: 1 (campaignAdditional Seal Battlefield Spoils Campaign)
+local isTradeNum2 = 0
+
+entity.onTrigger = function(player, npc)
+    local csid = 0x0160
```
Comment (claywar) [inline scripts/zones/Port_Jeuno/npcs/Shemo.lua]: Is there a reason that csid is stored as hex here with no manipulation, and then referenced as decimal in onEventFinish?

Comment (mikeyuzu) [reply scripts/zones/Port_Jeuno/npcs/Shemo.lua]: Unified csid to decimal numbers

**Review by claywar — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

**Pull request merged**